src/main.ts

```diff
@@ -1,21 +0,0 @@
-import Vue from 'vue';
```

Comment:
You should rename main.ts to app.ts, instead of removing/creating them, in my opinion.

Reply:
Nothing I can control, that's automatically determined by Vue :(

Reply:
Can't you use `git mv` for that?

Reply:
Afaik you can still use it after the original file is removed